File: C#_week 4/DateTimeCalculator/Models/OutputEntity.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace DateTimeCalculator.Models
{
    public class OutputEntity
    {
        public int Id { get; set; }

        public string Timestamp { get; set; }
        public DateTime Date { get; set; }
        public string TimeParams { get; set; }

        [Column(TypeName = "varchar(25)")]
        public DateOperation Operation { get; set; }

        public string Result { get; set; }

        public OutputEntity(string timestamp, DateTime date, string timeParams, DateOperation operation, string result)
        {
            this.Timestamp = timestamp;
            this.Date = date;
            this.TimeParams = timeParams;
            this.Operation = operation;
            this.Result = result;
            
        }
    }
}

File: week2&3/DateTimeCalculator/src/main/java/com/sapient/date_time_calculator/models/OutputEntity.java
package com.sapient.date_time_calculator.models;

import java.time.LocalDate;

public class OutputEntity {

	private LocalDate date;
	private int[] params;
	private DateOperation operation;
	private String result;

	public OutputEntity(LocalDate date, int[] params, DateOperation operation, String result) {
		super();
		this.date = date;
		this.params = params;
		this.operation = operation;
		this.result = result;
	}

	public LocalDate getDate() {
		return date;
	}

	public void setDate(LocalDate date) {
		this.date = date;
	}

	public int[] getParams() {
		return params;
	}

	public void setParams(int[] params) {
		this.params = params;
	}

	public DateOperation getOperation() {
		return operation;
	}

	public void setOperation(DateOperation operation) {
		this.operation = operation;
	}

	public String getResult() {
		return result;
	}

	public void setResult(String result) {
		this.result = result;
	}

}
